### ISSUE TITLE
cli requires case insensitive jar entries

### ISSUE THREAD

**Issue by kukgini** [cloudfoundry/cli#1147]
When I push app for java_buildpack. i got this message:
```
$ cf push
Using manifest file .../manifest.yml

Updating app <appname> in org <orgname> / space <spacename> as <appname>...
OK

FAILED
Error processing app files: mkdir /var/folders/51/r9kh5s8s6s.../T/unzipped-app006450987/META-INF/license: not a directory
```

There is case sensitive entries like this:
```
META-INF/LICENSE (which is file) 
META-INF/license/ 
META-INF/license/license-file-1.txt
META-INF/license/license-file-2.txt 
```
I am a MacOS X User (and sometimes use Windows of course). 
It seems like cf cli try to extract jar entries into /var/ before push it. 
But, MacOS default file system is case insensitive.

Sometimes, I can't manipulate jar entries because it is auto generated by build system (like gradle).
So, It's better not extracting files from jar.

**Comment by dkoper** [comment]
Thanks for submitting the issue.
We'd like to look at addressing it in our `push` refactor. I have prioritized the story accordingly.

Regards,
Dies Koper
CF CLI PM

**Comment by dkoper** [comment]
This is now resolved in cf CLI 6.34.0.

Regards,
Dies Koper
CF CLI PM
